src/resources/js/docs.js

```diff
@@ -15,11 +17,17 @@ $(function() {
 		// as another special case, highlight the modules link anywhere within it
 		$currentPageLink = $('main nav a[href="/docs/modules/"]');
 	}
-	$currentPageLink.addClass('current');
+	$currentPageLink.addClass("current");
 
 	// add anchor links, inspired by https://github.com/bryanbraun/anchorjs
-	$('article > h1[id], article > h2[id], article > h3[id], article > h4[id], article > h5[id], article > h6[id]').each(function() {
-		var $anchor = $('<a href="#'+this.id+'" class="anchor-link" title="Direct link">🔗</a>');
+	$(
+		"article > h1[id], article > h2[id], article > h3[id], article > h4[id], article > h5[id], article > h6[id]"
+	).each(function () {
```

Comment:
I assume you used Prettier? It does some truly weird stuff with line length. Not a fan. :man_shrugging:

Reply:
Yeah its enabled in my VSCode, sorry about that. I do find it helpful for keeping CSS/JS well formatted (with cavets). I should have checked with you guys first to see if it was an accepted practice. I'm onboard with your opinion that some of the prettier rules dont make much sense. I'll create a second PR with some sane Prettier standards so we can avoid further formatting issues going forwards.